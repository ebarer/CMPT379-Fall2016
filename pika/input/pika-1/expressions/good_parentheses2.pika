exec {
	const c := 3.1 - 4.27 * 5.001.
	const d := (3.1 - 4.27) * 5.001.
	const e := ((3.1 - 4.27) * 5.001).
	print c; d; e, _n_.
}
