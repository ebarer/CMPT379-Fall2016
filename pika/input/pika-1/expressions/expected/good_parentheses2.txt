-18.2543 -5.85117 -5.85117
